### DISCUSSION TITLE
The file table of the GitHub Repo page adds a heading attribute, which is redundant

### DISCUSSION THREAD

**Discussion by cary-rowen** [community/community#108106] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

For example:
https://github.com/nvaccess/nvda

When a user using a screen reader (such as NVDA) opens the page above, they may press T to navigate to the table, allowing the table to be read using table navigation commands.

When the user navigates the leftmost column "name", the user will find that each file has been added with "<h3>" attributes.
Is this intentional? Many people complain that this makes screen reader navigation less efficient. In the past, people were able to use H to quickly navigate to the readme section.
Now, it's hard for people to do that, 
Because every file has the "<h3>" attribute.

**Comment by ghostinhershell** [comment]
Hi @cary-rowen, 

Thanks so much for reaching out in our community. I'll go ahead and submit this feedback to the Accessibility team for review. If you have any additional feedback, details or reproduction steps that you would like to share, please feel free to comment below. Thanks again!

**Comment by ericwbailey** [comment, marked as answer]
@cary-rowen Hi there, and thank you for reporting this. I've just merged a PR to remove the `h3` heading elements from the Repository file table's row headers. 

You may need to refresh the tab for the changes to take effect, as well. Hopefully this will make navigation via heading a little quicker and easier.

**Comment by cary-rowen** [comment]
Thanks @ericwbailey 
This works for me.

However, I noticed that the "Latest commit" and "History" in the table still has the <h2> tag, is there any special purpose for this?

**Reply by ericwbailey** [reply]
@cary-rowen Glad to hear it works. The "Latest commit" and "History" `h2` subheadings are used to facilitate fast access to the last submitted commit to the repo, as well as access to the complete list of commits. 

Both are frequently accessed enough that it warranted placing a consistently-phrased text string and heading. The link names that follow them are dynamic, so it would make finding via <kbd>Control</kbd> + <kbd>F</kbd> difficult. Here we made the tradeoff of some extra verbosity to enable faster navigation.
